machine_learning_hep/utilities.py

```diff
@@ -871,10 +885,11 @@ def is_latex(obj):
         y_min_plot, y_max_plot = range_y
 
     # append "same" to the histogram plotting option if needed
-    opt_plot_h = opt_plot_h.lower()
-    opt_not_in = all(opt not in opt_plot_h for opt in ("same", "lego", "surf"))
-    if opt_not_in:
-        opt_plot_h += " same"
+    for opt in opt_plot_h:
+        opt = opt.lower()
+        opt_not_in = all(opt not in opt_plot_h for opt in ("same", "lego", "surf"))
+        if opt_not_in:
+                opt += " same"
```

Comment:
Same as above

Reply:
Looks like this has not been addressed.
It seems like you repeat the steps for characters in `opt_plot_h` in case it is `str`.